openrtb_ext/imp_resetdigital.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+type ExtResetDigital struct {
+	BidID string `json:"bid_id"`
+	ImpID string `json:"imp_id"`
+}
```

Comment:
`ExtResetDigital` is unused

Reply:
Removed on next commit.

Reply:
@bruno-siira  could can you link commit which removed `ExtResetDigital`